src/Module/Transaction/presentation/ListTransactions/useListTransaction.ts

```diff
@@ -38,25 +38,25 @@ export const useListTransaction = () : IUseListTransaction => {
         keyword: debouncedSearchTerm,
     });
     const navigation : StackNavigationProp<TransactionModuleStackParamList> = useTransactionNavigation();
-
+    console.log('queryListTransaction', queryListTransaction);
     const goToDetail = (transaction: ITransaction) => {
         navigation.navigate('DetailTransaction', { transaction });
     };
 
-    const onChangeKeyword = (text: string) => {
+    const onChangeKeyword = useCallback((text: string) => {
```

Comment:
adding useCallBack on functiion onChangeKeyboard and onSelectFilter, to avoid recreate a function